seqr/utils/elasticsearch/es_search.py

```diff
@@ -233,7 +234,8 @@ def filter_variants(self, inheritance=None, genes=None, intervals=None, rs_ids=N
         elif rs_ids:
             self._filter(Q('terms', rsid=rs_ids))
 
-        self._filter_by_frequency(frequencies, pathogenicity=pathogenicity)
+        clinvar_terms, hgmd_classes = _parse_pathogenicity_filter(pathogenicity or {})
+        self._filter_by_frequency(frequencies, clinvar_terms=clinvar_terms)
```

Comment:
is it intentional that _filter_by_frequency only uses clinvar_terms and not hgmd_classes?